### DISCUSSION TITLE
[Suggestion] Provide papi support for the placeholder of the essential chat format

### DISCUSSION THREAD

**Discussion by luoxiawuchen** [EssentialsX/Essentials#5223] [Ideas and Feature Suggestions] [answered]
With the development of the spigot/paper plugin ecosystem, papi has become the mainstream placeholder choice and has a large number of subordinate variables.

The placeholders provided by esschat are relatively limited and are not compatible with other plugins (such as various title plugins).
I hope to add the function of parsing papi variables to the chat format of esschat to make it more flexible.

Thank you for your patience.

**Comment by Chew** [comment, marked as answer]
https://github.com/EssentialsX/Essentials/issues/1945

Dunno if there's any PRs for this yet, I think JRoy is waiting on Adventure, but I'm not sure.

**Reply by luoxiawuchen** [reply]
> #1945
> 
> Dunno if there's any PRs for this yet, I think JRoy is waiting on Adventure, but I'm not sure.

Oh, thank you for your answer！
